### DISCUSSION TITLE
dev server error not shown

### DISCUSSION THREAD

**Discussion by Min5407** [quasarframework/quasar#16299] [General - Components / Directives / etc] [answered]
Hi, 

I have imported a function in a component which does not exsits but for some reason my dev server does not know that error has occured. Any way to display error message once I run dev server (yarn quasar dev) ??

![Screenshot 2023-09-05 at 11 32 22 AM](https://github.com/quasarframework/quasar/assets/46725533/4004254d-89ce-49e1-9c21-3f3c05b32124)
![Screenshot 2023-09-05 at 11 32 39 AM](https://github.com/quasarframework/quasar/assets/46725533/e0216ef9-a766-4795-acc9-1a9b645a4ef9)

**Comment by metalsadman** [comment]
https://stackoverflow.com/questions/72545700/vue-3-vite-typescript-dev-build-not-picking-up-ts-errors
https://vitejs.dev/guide/features.html#typescript

**Comment by rstoenescu** [comment, marked as answer]
There is a fundamental architecture difference between Webpack and Vite. While on dev, Vite parses on-the-fly only the files that handle the route currently being accessed through the browser. It does not compile your whole project, which is why the startup is so fast (among other reasons). However, this means that you need to build your project in order to detect possible issues with all your project's routes.
